filecache.go

```diff
@@ -16,13 +16,19 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// File is an abstraction to allow specifying an access token for the given file
+type File struct {
```

Comment:
I wouldn't call it `File` because it already confused me. Maybe call it `DownloadRecord` or something.

Reply:
That's brilliant, thanks!